src/CropTypeListBuilder.php

```diff
@@ -82,9 +96,25 @@ public function buildRow(EntityInterface $entity) {
     ];
     $row['description'] = Xss::filterAdmin($entity->description);
     $row['aspect_ratio'] = $entity->getAspectRatio();
+
+    // Load all image styles where the current crop type is used.
+    $usage = array();
+    $image_style_ids = $this->queryFactory->get('image_style')->condition('effects.*.data.crop_type', $entity->id())->execute();
```

Comment:
should be formatted correctly so the line isn't that long.
$this->queryFactory
  ->get()
  ...

This executes lots of entity queries if you have many crop types, I'm not sure if it would be more efficient to load the once on the plugin ID crop and then filter in PHP for each crop type. Note that there's e.g. no static cache for config entities, so it has to re-create those objects every time.

But, we already discussed that you probably won't have many crop types, so it shouldn't be a big deal.